blocks/to-do/add-and-delete-list-items.js

```diff
@@ -27,6 +33,9 @@ export default function addListItem(inputValue, list) {
     }
   };
 
-  // Add event listener
-  list.addEventListener('click', deleteListItem);
+  // Add only one event listener
+  if (list.getAttribute('data-has-listener') === 'false') {
+    list.addEventListener('click', deleteListItem);
+    list.setAttribute('data-has-listener', 'true');
+  }
```

Comment:
nice approach to get only a single listener! however, I'm still not convinced that adding an event listener to the *list* should be part of the function/file concerning itself with adding items.